src/Entity/SF1500/BrugerRegistreringAdresse.php

```diff
@@ -3,23 +3,18 @@
 namespace App\Entity\SF1500;
 
 use App\Repository\SF1500\BrugerRegistreringAdresseRepository;
-use App\Trait\IndeksTrait;
 use App\Trait\ReferenceIdTrait;
 use App\Trait\RolleTrait;
-use App\Trait\TypeTrait;
-use App\Trait\VirkningTrait;
 use Doctrine\ORM\Mapping as ORM;
 use Symfony\Component\Uid\Uuid;
 use Symfony\Component\Uid\UuidV4;
 
 #[ORM\Entity(repositoryClass: BrugerRegistreringAdresseRepository::class)]
+#[ORM\Index(columns: ['reference_id_uuididentifikator'], name: 'bruger_ref_uuid_idx')]
```

Comment:
Why isn't this index called `reference_id_uuididentifikator_idx`, i.e. the column name followed by `_idx`?